config/config.go

```diff
@@ -57,6 +57,12 @@ const (
 	// ElementsRPCEndpointKey is the url for the RPC interface of the Elements
 	// node in the form protocol://user:password@host:port
 	ElementsRPCEndpointKey = "ELEMENTS_RPC_ENDPOINT"
+	// ExplorerLimit represents number of requests per second that crawler
+	//makes to explorer
+	ExplorerLimitKey = "EXPLORER_LIMIT"
+	// ExplorerTokenBurst represents number of bursts tokens permitted from
+	//crawler to explorer
+	ExplorerTokenBurstKey = "EXPLORER_TOKEN_BURST"
```

Comment:
Maybe better to rename `CrawlerLimitKey` and `CrawlerTokenBurst` and relative values accordingly?

Reply:
Agreed, but would be consisten with CrawlIntervalKey therefore `CrawlLimitKey` and `CrawlTokenBurst`

Reply:
Done